sources/platform/actors/development/programming_interface/basic_commands.md

```diff
@@ -12,17 +12,18 @@ import TabItem from '@theme/TabItem';
 
 ---
 
-The commands described in this section are expected to be called from within a context of a running Actor, both in a local environment or on the Apify platform.
+This page covers essential commands for the Apify SDK in JavaScript & Python. These commands are designed to be used within a running Actor, either in a local environment or on the Apify platform.
 
-## Actor initialization
+## Initialize your Actor
 
-First, the Actor should be initialized. During initialization, it prepares to receive events from the Apify platform. The Actor determines the machine and storage configuration and optionally purges the previous state from local storage. It will also create a default instance of the Actor class.
+Before using any Apify SDK methods, initialize your Actor. This step prepares the Actor to receive events from the Apify platform, sets up machine and storage configurations, and optionally clears previous local storage states.
```

Comment:
the default storage clearing is opt-out, the default behavior is to clear them. this is something we changed with v3, and we are still missing this in the apify cli, there it is still opt-in - but we will be changing this to align the behavior of apify cli with the rest of the tooling in the next major version of the CLI (which is likely to be v1). currently, when you use `apify run`, the CLI effectively disables the auto purge that is otherwise handled by SDK

Reply:
so just removing `optionally` here  will be sufficient as right now this is default behaviour to purge the local states? But in the future once apify-cli moves to v1 it will become optional?

Reply:
it depends on what you are describing here, if its apify CLI usage, then right now, purging is still explicit, you need `-p`. but the behavior of SDK and crawlee are auto-purge, the CLI controls this explicitly via env vars. in CLI v1 we will unify the behavior so `apify run` will also auto-purge (and you will have an opt-out flag like `--no-purge`, with an alias `--resurrect`)

Reply:
This is SDK behaviour so it would so just removing `optionally` seems like easiest choice as it should be auto-purge